src/avt/IVP/avtCellLocator.C

```diff
@@ -56,18 +56,16 @@ avtCellLocator::SetDataSet(vtkDataSet *ds)
 {
     ReleaseDataSet();
     dataSet = ds;
-    dataSet->Register( NULL );
+    dataSet->Register( nullptr );
```

Comment:
Is `nullptr` a VTK-9 thing? Does this still work in VTK-8?

Reply:
It looks like our CI checks passed here so I think you are good to go.

Reply:
nullptr is C+11 so to explicitly set a pointer to null. Whereas NULL is just an int. 

https://dzone.com/articles/what-exactly-nullptr-is-in-c

As I touch code I have been replacing NULL with nullptr and at the same time doing inits (especially pointers) in the header file.